deploy/005-deploy-swaprouter.ts

```diff
@@ -5,10 +5,12 @@ const ADDRESSES = {
   bsctestnet: {
     WBNBAddress: "0xae13d989daC2f0dEbFf460aC112a837C89BAa7cd",
     pancakeFactory: "0x182859893230dC89b114d6e2D547BFFE30474a21",
+    comptrollerAddress: "0x94d1820b2D1c7c7452A163983Dc888CEC546b77D",
```

Comment:
The Comptroller is deployed in the same package. Would it be more flexible if we recover the Comptroller address dynamically, as we are doing [here](https://github.com/VenusProtocol/venus-protocol/blob/develop/deploy/004-support-markets.ts#L12)

Reply:
Let's do these changes in separate PR. As these fixes are related to the swap audits.

Reply:
Fixed.